ping.py

```diff
@@ -402,10 +517,10 @@ def verbose_ping(hostname, timeout=3000, count=3,
     yield not myStats.pktsRcvd
 
 
-def quiet_ping(hostname, timeout=3000, count=3,
+def quiet_ping(hostname, timeout=3000, count=3, advanced_statistics=False,
```

Comment:
Should the advanced statistics not have an access from the argparse command line?